code/drasil-example/glassbr/lib/Drasil/GlassBR/ModuleDefs.hs

```diff
@@ -185,7 +189,7 @@ interpY = funcDef "interpY"
   ]
 
 interpZ :: Func
-interpZ = funcDef "interpZ" 
+interpZ = funcDef (showUID U.interpZ)
```

Comment:
To continue the discussion we had on Teams here a bit, `showUID` (something that shouldn't really exist outside of ChunkDB debugging code) usage here is only desired because we want it to rely on the information from the newly created QuantityDicts (I guess a more appropriate way to get the function name would be a "showSymbol"-like function for the implementation-stage name of the interpZ QuantityDict), but this _should_ be best "cleaned up" if/when we place `Func`s inside of the ChunkDBs. As they are right now, we aren't able to palce them in the ChunkDBs because (a) they don't have UIDs, and (b) the ChunkDBs would require a bit more effort for us to be able to palce them inside of them (maybe #2873).

Reply:
Using a hacked up `showSymbol`-like function would be quite a bit superior to using `showUID`. It would at least be pulling from the 'right' bucket of information.

Reply:
I agree, that makes sense.

Regarding the type of the symbol, should it really be a String? Specifically, is there any worth in changing it to the "Implementation-level Symbol"?

If we choose String, I think we have ~2 options:
1) figure out a way to convert the "Implementation-level Symbol" into a Doc/Text (this should already exist somewhere I believe), and then into extract it into a String, or
2) roll our own "Symbol -> String" function (of course, this creates potential room for misalignment with the existing Symbol -> Doc/Text functions)

Reply:
No, it doesn't have to be String. Thinking about it, it should indeed be implementation-level symbol. Well spotted!